aom/src/main/java/com/nedap/archie/adlparser/treewalkers/ComponentTerminologiesHelper.java

```diff
@@ -0,0 +1,22 @@
+package com.nedap.archie.adlparser.treewalkers;
+
+import com.nedap.archie.aom.terminology.ArchetypeTerminology;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+/**
+ * A helper class for parsing component terminologies. Not to be used outside of the parser
+ */
```

Comment:
So make it package-private? This obviously isn't possible if the parser consists of multiple packages, but I am not sure if it is

Reply:
I'll change it.